sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -16,59 +16,83 @@ import TabItem from '@theme/TabItem';
-| `ACTOR_WEB_SERVER_URL`             | A unique public URL under which the Actor run web server is accessible from the outside world.                                                                                                                                                         |
+Apify sets several system environment variables for each Actor run. These variables provide essential context and information about the Actor's execution environment.
+
+Here's a table of key system environment variables:
+
+| Environment Variable | Description |
+|----------------------|-------------|
+| `ACTOR_ID` | ID of the Actor. |
+| `ACTOR_RUN_ID` | ID of the Actor run. |
+| `ACTOR_BUILD_ID` | ID of the Actor build used in the run. |
+| `ACTOR_BUILD_NUMBER` | Build number of the Actor build used in the run. |
+| `ACTOR_TASK_ID` | ID of the Actor task. Empty if Actor is run outside of any task, e.g. directly using the API. |
+| `ACTOR_EVENTS_WEBSOCKET_URL` | Websocket URL where Actor may listen for events from Actor platform. |
+| `ACTOR_DEFAULT_DATASET_ID` | ID of the dataset where you can push the data. |
+| `ACTOR_DEFAULT_KEY_VALUE_STORE_ID` | ID of the key-value store where the Actor's input and output data are stored. |
+| `ACTOR_DEFAULT_REQUEST_QUEUE_ID` | ID of the request queue that stores and handles requests that you enqueue. |
+| `ACTOR_INPUT_KEY` | Key of the record in the default key-value store that holds the Actor input. Typically **INPUT**, but may vary. |
+| `ACTOR_MAX_PAID_DATASET_ITEMS` | For paid-per-result Actors, the user-set limit on returned results. Do not exceed this limit. |
+| `APIFY_HEADLESS` | If **1**, web browsers inside the Actor should run in headless mode (no windowing system available). |
+| `APIFY_IS_AT_HOME` | Set to **1** if the Actor is running on Apify servers. |
+| `ACTOR_MEMORY_MBYTES` | Size of memory allocated for the Actor run, in megabytes. Used to optimize memory usage. |
+| `APIFY_PROXY_PASSWORD` | The Apify Proxy password of the user who started the Actor. |
+| `APIFY_PROXY_PORT` |  |
+| `APIFY_PROXY_STATUS_URL` |  |
+| `ACTOR_STANDBY_PORT` | TCP port for the Actor to start an HTTP server to receive messages from Actor Standby. |
+| `ACTOR_STARTED_AT` | Date when the Actor was started. |
+| `ACTOR_TIMEOUT_AT` | Date when the Actor will time out. |
+| `APIFY_TOKEN` | API token of the user who started the Actor. |
+| `APIFY_USER_ID` | ID of the user who started the Actor. May differ from the Actor owner. |
+| `ACTOR_WEB_SERVER_PORT` | TCP port for the Actor to start an HTTP server to receive external messages. |
+| `ACTOR_WEB_SERVER_URL` | Unique public URL for accessing the Actor run web server from the outside world. |
+| `APIFY_API_PUBLIC_BASE_URL` | Public URL of the API, typically set to api.apify.com |
+| `APIFY_DEDICATED_CPUS` | Number of CPU cores reserved for the actor, based on allocated memory. |
+| `APIFY_DISABLE_OUTDATED_WARNING` |  |
+| `APIFY_FACT` |  |
```

Comment:
```suggestion
| `APIFY_FACT` | Easter egg |
```

We may just leave this one out I guess :slightly_smiling_face:

Reply:
Best to remove that if nothing of value is added :D

Reply:
Yeah, I think that if somebody can't understand it's an easter egg when they encounter it, they're lost anyway.